libs/designsystem/src/lib/components/chart/chart.component.ts

```diff
@@ -22,12 +23,16 @@ import { ACTIVITYGAUGE_OPTIONS, ActivityGaugeOptions } from './options/activityg
 })
 export class ChartComponent implements OnInit, OnChanges {
   @Input() data = [];
-  @Input() breaks: Array<Highcharts.XAxisBreaksOptions> = [];
+  @Input() breaks: Array<XAxisBreaksOptions> = [];
   @Input() height = 300;
   @Input() type: ChartType = ChartType.PIE;
   @Input() description = '';
   @Input() showDataLabels = true;
-  options: Options = {};
+  // tslint:disable-next-line: no-input-rename
+  @Input('options') optionsForOverride: Options;
```

Comment:
I think it is more readable to call this `optionsForOverride` internally and have it as `options` in the external interface.

Reply:
I don't :)
Let's turn that around, so it's called options and the other one mergedOptions or similar...

Reply:
Fixed the naming so, input is options and internal is mergedOptions